awx/main/models/mixins.py

```diff
@@ -444,3 +447,36 @@ def clean_custom_virtualenv(self):
         if value:
             return os.path.join(value, '')
         return None
+
+
+class RelatedJobsMixin(object):
+
+    '''
+    Implement me
```

Comment:
I think I would rather see the how to implement here instead of in the exception message.